zeyple/zeyple.py

```diff
@@ -5,8 +5,10 @@
 import os
 import logging
 import email
+import email.mime.multipart
 import smtplib
 import gpgme
+import copy
 from io import BytesIO
 try:
     from configparser import SafeConfigParser  # Python 3
```

Comment:
Not part of your changes but shouldn't we use `from six.moves.configparser import ConfigParser` instead here?